Dockerfile

```diff
@@ -12,12 +12,13 @@ RUN apk --no-cache add \
     boost-dev \
     libxml2-dev \
     libuv-dev \
-    util-linux-dev
+    util-linux-dev \
+    perl
```

Comment:
Where does the perl requirement come from? Is this a consequence of package upgrade or do we have a new dependency?